paddle/fluid/inference/api/analysis_predictor_tester.cc

```diff
@@ -260,7 +260,8 @@ class MkldnnQuantizerTest : public testing::Test {
     predictor.reset(new AnalysisPredictor(config));
     auto* predictor_p = static_cast<AnalysisPredictor*>(predictor.get());
 
-    auto qconfig = std::make_shared<MkldnnQuantizerConfig>();
+    // The MkldnnQuantizerConfig will be deleted by MkldnnQuantizer's destructor
```

Comment:
Hey, this comment is no longer necessary. :)

Reply:
done